contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -7,14 +7,18 @@ import {IAggregator} from "./interfaces/IAggregatorSubmit.sol";
 // TODO: submission verification
 // TODO: submission by aggregator name
 contract SubmissionProxy is Ownable {
+    uint256 public constant MIN_SUBMISSION = 0;
+    uint256 public constant MAX_SUBMISSION = 1_000;
+    uint256 public constant MIN_EXPIRATION = 1 days;
+    uint256 public constant MAX_EXPIRATION = 365 days;
+
     uint256 public maxSubmission = 50;
     uint256 public expirationPeriod = 5 weeks;
 
-    address[] public oracles;
-    mapping(address => uint256) expirations;
+    // oracle => expiration
+    mapping(address => uint256) oracles;
```

Comment:
it's optional, how about removing `// oracle => expiration` comment and updating into 
```
mapping(address oracleAddress => uint256 expiration) oracles
```
solidity supports named mapping
reference: https://soliditylang.org/blog/2023/02/01/solidity-0.8.18-release-announcement/

Reply:
Looks great!